lib/src/picture_provider.dart

```diff
@@ -299,7 +305,7 @@ class PictureConfiguration {
 abstract class PictureProvider<T> {
   /// Abstract const constructor. This constructor enables subclasses to provide
   /// const constructors so that they can be used in const expressions.
-  const PictureProvider(this.colorFilter);
+  PictureProvider(this.colorFilter);
```

Comment:
We should check if this will break any internal clients.

Reply:
Do you know if there is an easy way to validate that? Regarding the usage of `HttpClient`, from what I see, a new instance is created in each `PictureProvider._loadAsync` call.

Reply:
I did a quick internal code search and we have one client this would break. It should be easy to fix.